CMakeLists.txt

```diff
@@ -233,6 +233,8 @@ if(WIN32)
   if(NOT SEQUENTIAL_LINK_LINE) 
     set(SEQUENTIAL_LINK_LINE ${DEFAULT_WIN32_SEQUENTIAL_LINK_LINE} CACHE STRING "Arguments passed to compiler to link host libraries to SYCL applications")
   endif()
+  # Clang on Windows adds invalid padding to accessor structs, workaround by using standard (inefficient) accessors
+  set(CUDA_EXTRA_CXX_FLAGS "-DHIPSYCL_STRICT_ACCESSOR_DEDUCTION")
```

Comment:
Maybe `CUDA_PLATFORM_SPECIFIC_CXX_FLAGS` or similar - I believe that's the intention?